examples/java/core/serialization/src/main/bond/serialization.bond

```diff
@@ -0,0 +1,8 @@
+namespace examples.serialization
```

Comment:
This would be inappropriate for Javaland because it's not a domain we control. I would say that if you're writing a bondfile specifically for Java consumption, your namespace should match Java conventions (e.g., `com.microsoft.bond.examples.serialization`) and if not, you should use a mapping (see `java/core/build.gradle`). I'd rather see this example do one of those two things. Any thoughts?